pages/cart-brand-connections.js

```diff
@@ -0,0 +1,99 @@
+import Head from 'next/head'
+import styles from 'css/Cart.module.css'
+import { publicRuntimeConfig } from 'next.config';
+import router from 'next/router';
+import { useEffect, useState } from 'react';
+import Script from "next/script";
+
+function CartBrandConnections() {
+  const basePath =  router?.router?.basePath || publicRuntimeConfig.basePath;
+  const [member_id, setMemberId] = useState('');
+  const [price, setPrice] = useState(262.94);
+  const [discountedPrice, setDiscountedPrice] = useState(price);
+  const [userLoaded, setUserLoaded] = useState(false);
+  const [userinfo, setUserinfo] = useState();
+
+  useEffect(() => {
+    (async () => {
+      setUserLoaded(true);
+      setMemberId(userinfo?.member_id);
+      if (userinfo?.member_id) {
+        let newPrice = Number(price * .9).toFixed(1).toLocaleString();
+        setDiscountedPrice(newPrice);
+      }
+    })()
+  }, [userinfo]);
+
+  return (
+    <div className={styles.container}>
+    <Head>
+        <title>Checkout Your Cart</title>
+        <meta name="description" content="Demo of WeSalute Login" />
+        <link rel="icon" href="/favicon.ico" />
+    </Head>
+      <Script
+        src="https://va-brand-connections-loader-dev.firebaseapp.com/index.js"
+        onReady={() => {
+          initiateVABrandConnectionsComponent({
+            clientId: 'd73055df-24a9-4313-9968-0cdc99934d9e',
+            premium: true,
+            userInfo: function (data, error) {
+              if (error) {
+                console.log("error", error);
+              } else if (data) {
+                console.log("userInfo", data);
+                setUserinfo(data);
+              }
```

Comment:
`else if` assumes there may be another option, which is not handled here, so this should be just `else`.